services/rolemanager.md

```diff
@@ -1,6 +1,6 @@
 # Role Manager
 
-    identifier: 0x119c3ad1
+    identifier: 0x1
```

Comment:
typo?

Reply:
kind of, I just want a new identifier here - that's a way for the spectool to suggest one